3-networks/modules/base_shared_vpc/firewall.tf

```diff
@@ -69,121 +69,3 @@ resource "google_compute_firewall" "allow_private_api_egress" {
 
   target_tags = ["allow-google-apis"]
 }
-
-
-/******************************************
-  Optional firewall rules
- *****************************************/
-
-// Allow SSH via IAP when using the allow-iap-ssh tag for Linux workloads.
-resource "google_compute_firewall" "allow_iap_ssh" {
-  count   = var.optional_fw_rules_enabled ? 1 : 0
```

Comment:
In shared can we continue to support flags like `optional_fw_rules_enabled`? Perhaps a `merge` on `rules` map for `allow-iap-ssh-rdp` etc?

Reply:
We can do that but it means all environments get the same policy (i.e. can't have an environment without the optional rules and another environment with them)

What do you think?

Reply:
I see, in that case perhaps these should be regular FW rules which we can control per env and can override inherited HFW rules. @rjerrems WDYT?

Reply:
I think that makes sense to me